torch/_ops.py

```diff
@@ -628,11 +628,13 @@ def unset_mode_pre_dispatch(mode_key, schema_check=False):
         assert mode_key is None
 
     def _unset_mode():
-        if mode_key == torch._C._TorchDispatchModeKey.PROXY:
+        # NOTE: Using `is` rather than `==` to work around slow enum comparison in
```

Comment:
Maybe this isn't a good place for this comment, particularly since you have to repeat it below!

Perhaps it would be better in the documentation of `torch._C._TorchDispatchModeKey` where others would see it?